src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java

```diff
@@ -847,6 +884,160 @@ public void uploadWithScheduleContext() throws Exception {
         UploadTableTest.assertCsvFile(csvFile, SURVEY_UPLOAD_TABLE_COLUMNS, expectedRowsMap);
     }
 
+    @Test
+    public void uploadArcResultWithScheduleContext() throws Exception {
+        TestUser admin = TestUserHelper.getSignedInAdmin();
+        AssessmentsApi assessmentsApi = admin.getClient(AssessmentsApi.class);
+        ParticipantsApi participantsApi = admin.getClient(ParticipantsApi.class);
+        SchedulesV2Api schedulesApi = admin.getClient(SchedulesV2Api.class);
+        StudiesApi studiesApi = admin.getClient(StudiesApi.class);
+        UploadsApi uploadsApi = admin.getClient(UploadsApi.class);
+
+        // Get the user's healthcode.
+        String userId = user.getUserId();
+        StudyParticipant participant = participantsApi.getParticipantById(userId, false).execute().body();
+        String healthCode = participant.getHealthCode();
+
+        // Give participant  data group.
+        participant.dataGroups(ImmutableList.of(DATA_GROUP_SDK_INT_1));
+        participantsApi.updateParticipant(userId, participant).execute();
+
+        // Make Open Bridge survey assessment.
+        String assessmentId = "symbol_test";
+
+        // Get ARC symbol test
+        Assessment assessment = assessmentsApi.getAssessmentById(assessmentId, 1L).execute().body();
+
+        // To run locally you will need to create a symbol_test assessment
+        //if (assessment == null) {
```

Comment:
The assessments created by this test are deleted in after(), so we'll need to create them every time.

If we want to have assessments that stick around across integ test runs, we'll want to add them to https://github.com/Sage-Bionetworks/BridgeIntegrationTests/blob/develop/src/test/java/org/sagebionetworks/bridge/sdk/integration/InitListener.java (This runs in the Server Integ Tests.) Long term, we're going to use the BridgeStackBuilder (under construction) to boot strap test environments.

Reply:
As per Wednesday's meeting, these tests don't run on Prod, so we don't worry about cluttering up Prod with test assessments.

In Dev/Staging, we can either create and delete the assessment with every test run, or we can check whether the assessment already exists and create it if it doesn't, which should leave a singleton assessment that persists across test runs. (Your commented out code already does this.)